.storybook/preview.js

```diff
@@ -21,5 +21,9 @@ export const decorators = [
       </MemoryRouter>
```

Comment:
It looks like `ThemeProvider` has moved to the theme provider. Might need to remove `<ThemeProvider>` as.